librz/include/rz_util/set.h

```diff
@@ -14,6 +14,7 @@ extern "C" {
 typedef HtPP SetP;
 
 RZ_API SetP *set_p_new(void);
```

Comment:
i don't even know what this is and why we have them nor where we use them.

Reply:
and btw they should have `rz_`... these definitely needs some refactoring!

Reply:
btw, `set_p` does not seem to be used anywhere and i'm not sure it would really make sense to have, unless we define a very specific set (e.g. a set with strings as key). Otherwise, we are probably better of by leaving the developer with the choice of the type of hashtable he wants. 

IMHO we can remove `set_p`.